near-sdk/src/json_types/public_key.rs

```diff
@@ -1,19 +1,29 @@
-use borsh::{BorshDeserialize, BorshSerialize};
+use borsh::{maybestd::io, BorshDeserialize, BorshSerialize};
```

Comment:
prob not best to use `maybestd` from borsh, as how borsh handles nostd might be very different than us in the future and ties us to that dependency

Reply:
I see, that's a good point. Maybe unwrapping the results and re-wrapping it in our own Result type would do the trick?